### ISSUE TITLE
HttpOnly cookies

### ISSUE THREAD

**Issue by ianwalter** [ianwalter/bouncer#20]
Document how to use HttpOnly cookies with Bouncer (and make it easier to do so if necessary) to prevent XSS attacks as described in http://www.redotheweb.com/2015/11/09/api-security.html.

**Comment by exploreitspace** [comment]
Stumbled upon your project @ianwalter, like the idea behind the project. Are you still conisidering maintaining this project and what sort of help you need here! are you planning to implement something similar to `Plug.Session` cookie implementation here.

**Comment by ianwalter** [comment]
Yes, I still want to implement this feature, but it's not very high on the priority list so I don't know when it will land. PRs welcome!
